package.json

```diff
@@ -111,10 +111,10 @@
     "@electron-forge/maker-squirrel": "6.0.0-beta.54",
     "@electron-forge/plugin-webpack": "6.0.0-beta.54",
     "@marshallofsound/webpack-asset-relocator-loader": "0.5.0",
-    "@storybook/addon-actions": "^6.2.9",
-    "@storybook/addon-essentials": "^6.2.9",
-    "@storybook/addon-links": "^6.2.9",
-    "@storybook/react": "^6.2.9",
+    "@storybook/addon-actions": "6.2.9",
+    "@storybook/addon-essentials": "6.2.9",
+    "@storybook/addon-links": "6.2.9",
+    "@storybook/react": "6.2.9",
```

Comment:
Same concern as above, it was mistake to merge this commit(adding storybook) into master, but no panic because it does nothing actually.

Reply:
Yes, I know it is an update to remove `^` to make the package.json consistent.